megatron/core/models/mamba/mamba_model.py

```diff
@@ -74,6 +86,8 @@ def __init__(
         scatter_embedding_sequence_parallel: bool = True,
         seq_len_interpolation_factor: Optional[float] = None,
         pg_collection: Optional[ProcessGroupCollection] = None,
+        vp_stage: Optional[int] = None,
+        mtp_block_spec: Optional[ModuleSpec] = None,
```

Comment:
Why do we have a separate spec for the MTP block? Why is this not part of the hybrid model spec?